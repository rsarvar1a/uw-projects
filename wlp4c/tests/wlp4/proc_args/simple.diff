11c11
< $03 = 0x000001bc
---
> $03 = 0x000000ec
