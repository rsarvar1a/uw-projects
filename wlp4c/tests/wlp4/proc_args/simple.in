3
51 39 22

