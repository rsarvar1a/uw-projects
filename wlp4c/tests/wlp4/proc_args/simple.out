Enter length of array: Enter array element 0: Enter array element 1: Enter array element 2: Running MIPS program.
51
52
51
39
22
51
39
22
MIPS program completed normally.
$01 = 0x00000a34   $02 = 0x00000003   $03 = 0x000001bc   $04 = 0x00000004   
$05 = 0x00000010   $06 = 0x00000000   $07 = 0x00000164   $08 = 0x00000008   
$09 = 0x00000a34   $10 = 0x00000000   $11 = 0x00000001   $12 = 0x00000000   
$13 = 0x00000000   $14 = 0x00000000   $15 = 0x00000000   $16 = 0x00000000   
$17 = 0x00000000   $18 = 0x00000000   $19 = 0x00000000   $20 = 0x00000000   
$21 = 0x00000000   $22 = 0x00000000   $23 = 0x00000000   $24 = 0x00000000   
$25 = 0x00000000   $26 = 0x00000000   $27 = 0x00000000   $28 = 0x00000310   
$29 = 0x01000000   $30 = 0x01000000   $31 = 0x8123456c   
